be/CMakeLists.txt

```diff
@@ -301,6 +303,33 @@ set_target_properties(opentelemetry_resources PROPERTIES IMPORTED_LOCATION ${THI
 add_library(opentelemetry_exporter_jaeger_trace STATIC IMPORTED GLOBAL)
 set_target_properties(opentelemetry_exporter_jaeger_trace PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib64/libopentelemetry_exporter_jaeger_trace.a)
 
+if ("${USE_STAROS}" STREQUAL "ON")
+     find_package(protobuf CONFIG REQUIRED)
+     message(STATUS "Using protobuf ${protobuf_VERSION}")
+     message(STATUS "protobuf include path: ${protobuf_INCLUDE_DIRS}")
+     message(STATUS "protobuf libs: ${protobuf_LIBS}")
+
+     find_package(gRPC CONFIG REQUIRED)
+     message(STATUS "Using gRPC ${gRPC_VERSION}")
+     get_target_property(gRPC_INCLUDE_DIR gRPC::grpc INTERFACE_INCLUDE_DIRECTORIES)
+     include_directories(${gRPC_INCLUDE_DIR})
+
```

Comment:
If Starlet was built with Protobuf and gRPC, there should be no need to add dependency of them here.

Reply:
it will causes compile err if you remove these find package

Reply:
If so, there may be some problems with the installation of starlet.